subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1437,7 +1573,7 @@ bool GameLogicPluginPrivate::OnNewArtifact(const subt::msgs::Artifact &_req,
   {
     std::lock_guard<std::mutex> lock(this->mutex);
     auto [scoreDiff, duplicate] = this->ScoreArtifact(
-        artifactType, _req.pose());
+        artifactType, _req.pose(), artifactMsg);
```

Comment:
Can we avoid publishing duplicate reports to the `/subt/artifact_reports` topic? Currently, when a duplicate report is received, this publishes a message with only timestamp. @nkoenig let me know if you want the example rosbag.

Reply:
I believe @angelacmaio fixed this.